apps/authentication/index.coffee

```diff
@@ -0,0 +1,23 @@
+express = require 'express'
+routes = require './routes'
+middleware = require './middleware'
+{ loginPath } = require('arena-passport').options
+
+app = module.exports = express()
+
+app
+  .set 'views', "#{__dirname}/templates"
+  .set 'view engine', 'jade'
+
+app
+  # TODO: None of these paths make any kind of sense (snake_case;
+  # language: "sign up" vs "join"). Generate new routes and redirect to old.
+  .get '/sign_up', routes.sign_up
+  .get '/log_in', routes.log_in
+  .post loginPath, middleware.return # /me/sign_in
```

Comment:
Maybe now is a good time to make these routes match the language we use. `sign_up => join`, `log_in => log-in`, etc...